public/user_img.html

```diff
@@ -42,13 +42,13 @@
         <div class="collapse navbar-collapse" id="navbarResponsive">
           <ul class="navbar-nav ml-auto">
             <li class="nav-item mx-0 mx-lg-1">
-              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#portfolio">Home</a>
+              <a href="index.html" class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger">Home</a>
             </li>
             <li class="nav-item mx-0 mx-lg-1">
-              <a class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger" href="#about">LOGIN</a>
+              <a href="login.html" class="nav-link py-3 px-0 px-lg-3 rounded js-scroll-trigger">LOGIN</a>
```

Comment:
In #41 I proposed keeping this as "/signup" for now but I'm happy to have it link to a "/login" route that sends back `login.html`. Current is fine though I suppose